### ISSUE TITLE
SI Hackathon: Add OSIA_UIN_VID_NID as a FHIR Identifier in Event Notification API

### ISSUE THREAD

**Issue by euanmillar** [opencrvs/opencrvs-core#5110]
Ensure that the FHIR identifier OSIA_UIN_VID_NID can be submitted in the Event Notification payload for any individual.  It should be saved into the FHIR Patient identifier array

**Comment by Zangetsu101** [comment]
@euanmillar Now that I think about it, I think this should work out of the box after #5044 merges in. I missed the fact that the OSIA_UIN_VID_NID will be an actual field in our form (the national id field)

**Comment by Zangetsu101** [comment]
We could provide an example postman collection with the OSIA_UIN_VID_NID tho

**Comment by naftis** [comment]
@euanmillar I tested with the Event notification Postman collection in Farajaland and can confirm what @Zangetsu101 already pointed out. I added 
```
{
    "use": "en",
    "type": "OSIA_UIN_VID_NID",
    "value": "6848901132"
}
```
to fathers and mothers identifiers and they appear similarly in Mongo hearth-dev Patient collection as when verifying with NID when doing the form. 

Additionally, to the object can be added a key to 'lock in' the verified values that OSIA/MOSIP has provided, to disallow editing them. Don't think this is necessary with the event notification:

```
{
    "use": "en",
    "type": "OSIA_UIN_VID_NID",
    "value": "6848901132",
    "fieldsModifiedByIdentity": "iD,motherBirthDate,firstNamesEng,familyNameEng"
}
```
